o1vm/src/mips/tests.rs

```diff
@@ -297,8 +284,7 @@ mod folding {
         let mut rng = o1_utils::tests::make_test_rng();
         // We only care about instruction parts and instruction pointer
         let mut dummy_env = dummy_env(&mut rng);
-        // Instruction: 0b10001111101001000000000000000000
-        // addi	a1,sp,4
+        // Instruction: 0b10001111101001000000000000000000 addi a1,sp,4
```

Comment:
I would leave it as before.

Reply:
stupid command I found for vscode to limit 80 chars...